docs/static/includes/interfaces/int.pe.udt.schema2.bicep

```diff
@@ -95,7 +95,7 @@ module <singularMainResourceType>_privateEndpoints 'br/public:avm/res/network/pr
     privateDnsZoneResourceIds: privateEndpoint.?privateDnsZoneResourceIds
     roleAssignments: privateEndpoint.?roleAssignments
     tags: privateEndpoint.?tags ?? tags
-    manualPrivateLinkServiceConnections: privateEndpoint.?manualPrivateLinkServiceConnections
+    manualPrivateLinkServiceConnections: !empty(privateEndpoint.?privateLinkServiceConnections) ? [] : privateEndpoint.?manualPrivateLinkServiceConnections
```

Comment:
```suggestion
    manualPrivateLinkServiceConnections: privateEndpoint.?manualPrivateLinkServiceConnections ?? []
```

Reply:
Don't you mean `manualPrivateLinkServiceConnections: privateEndpoint.?privateLinkServiceConnections ?? []`?
I want `manualPrivateLinkServiceConnections` to be used only when `privateLinkServiceConnections` is `[]`.

Reply:
Anyway, adding pre-constructed payload as per https://github.com/Azure/bicep-registry-modules/issues/946.

Reply:
As above, this should've been

```bicep
manualPrivateLinkServiceConnections: privateEndpoint.?manualPrivateLinkServiceConnections ?? []
```